content/en/docs/v3.3/installing-on-linux/cluster-operation/add-edge-nodes.md

```diff
@@ -24,9 +24,35 @@ This tutorial demonstrates how to add an edge node to your cluster.
 - You have an available node to serve as an edge node. The node can run either Ubuntu (recommended) or CentOS. This tutorial uses Ubuntu 18.04 as an example.
 - Edge nodes, unlike Kubernetes cluster nodes, should work in a separate network.
 
+## Evict non-edge workloads
+
+Due to the tolerations some daemonsets (for example, Calico) have, to ensure that the newly added edge nodes work properly, you need to run the following command to manually patch the pods so that non-edge workloads will not be scheduled to the edge nodes.
+
+```bash
+#!/bin/bash
+   
+   
+NoShedulePatchJson='{"spec":{"template":{"spec":{"affinity":{"nodeAffinity":{"requiredDuringSchedulingIgnoredDuringExecution":{"nodeSelectorTerms":[{"matchExpressions":[{"key":"node-role.kubernetes.io/edge","operator":"DoesNotExist"}]}]}}}}}}}'
+   
+ns="kube-system"
+
+
+DaemonSets=("nodelocaldns" "kube-proxy" "calico-node")
+
+length=${#DaemonSets[@]}
+   
+for((i=0;i<length;i++));  
+do
+         ds=${DaemonSets[$i]}
+        echo "Patching resources:DaemonSet/${ds}" in ns:"$ns",
+        kubectl -n $ns patch DaemonSet/${ds} --type merge --patch "$NoShedulePatchJson"
+        sleep 1
+done
+```
+
 ## Configure an Edge Node
```

Comment:
`## Configure an Edge Node` should be moved to a position between `## Create Firewall Rules and Port Forwarding Rules` and `## Add an Edge Node`